blob/src/main/java/io/crate/blob/BlobTransferTarget.java

```diff
@@ -107,14 +111,20 @@ public void startTransfer(StartBlobRequest request, StartBlobResponse response)
         response.size(digestBlob.size());
         if (request.isLast()) {
             try {
-                digestBlob.commit();
+                digestBlob.commit(blobCoordinator);
```

Comment:
why not hold the coordinator in the digestBlob? this looks to me like an internal thing of digestblob